pkg/morph/client/notary.go

```diff
@@ -586,6 +597,17 @@ func (c *Client) notaryWitnesses(invokedByAlpha bool, multiaddr *wallet.Account,
 		VerificationScript: multiaddr.GetVerificationScript(),
 	})
 
+	// then we have invoker witness
+	invokeScript = append(
```

Comment:
Use `c.acc.SignTx(...)` instead of subsequent 10 lines of code. It does exactly the same thing.